src/ci/scripts/install-msys2-packages.sh

```diff
@@ -6,7 +6,8 @@ IFS=$'\n\t'
 source "$(cd "$(dirname "$0")" && pwd)/../shared.sh"
 
 if isWindows; then
-    pacman -S --noconfirm --needed base-devel ca-certificates make diffutils tar
+    pacman -S --noconfirm --needed base-devel ca-certificates make diffutils tar \
+        binutils
```

Comment:
It's weird, they are provided by `mingw-w64` toolchain (in a version that really works for Windows).